launch/openHAB2.setup

```diff
@@ -2430,7 +2430,157 @@
     <stream
         name="master"/>
   </project>
-  <project name="4_esh"
+  <project name="4_zigbee"
+      label="openHAB ZigBee binding">
```

Comment:
Could you change all occurrences of "binding" to "binding" to be in sync with the other texts (where we also have e.g. "Add-ons")?

Reply:
From "binding" to "binding" would be a no-op ;) I guess you mean the capitalization, as this is the inconsistency I could spot; updated the PR by capitalizing "binding" for both installer labels and working set labels.

Reply:
You are certainly very right 🤣